static/app/views/replays/detail/page.tsx

```diff
@@ -46,6 +48,15 @@ export default function Page({
   const onShareReplay = useShareReplayAtTimestamp();
   const onDeleteReplay = useDeleteReplay({replayId: replayRecord?.id, projectSlug});
 
+  const {replay} = useReplayContext();
+  const rrwebFrames = replay?.getRRWebFrames();
+  // The replay data takes a while to load in, which causes our `isVideoReplay`
+  // to return an early `false`, which used to cause UI jumping.
+  // One way to check whether it's finished loading is by checking the length
+  // of the rrweb frames, which should always be > 2 for any given replay.
+  // By default, the 2 frames are replay.start and replay.end
+  const isLoading = !rrwebFrames || (rrwebFrames && rrwebFrames.length <= 2);
```

Comment:
Can we hoist this variable up into static/app/views/replays/details.tsx and pass it down? It's similar in my mind to `isVideoReplay` and should be set together.

Eventually this logic could be returned from `useReplayData`, so it can be re-used